packages/shorebird_cli/lib/src/commands/patch/patch_android_command.dart

```diff
@@ -78,6 +79,22 @@ If this option is not provided, the version number will be determined from the p
         'staging',
         negatable: false,
         help: 'Whether to publish the patch to the staging environment.',
+      )
+      ..addFlag(
+        'sign',
+        help: r'''
+When true will sign allow the hash of the patch to be signed.
```

Comment:
Do we need to ship this code? It seems like our users would not benefit from this so is there any harm to keeping this on a branch for our testing?

Reply:
I don't have a strong opinion. I foresee that part of the changes of this PR will not change. But I see no issues in keeping this open and use it from the branch while we are working on the patch sigining feature.